shortfin/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ add_compile_options("$<$<C_COMPILER_ID:MSVC>:/utf-8>")
 add_compile_options("$<$<CXX_COMPILER_ID:MSVC>:/utf-8>")
 
 # Pins
+# Keep in sync with requirements-iree-compiler.txt.
```

Comment:
Update this if moving the requirements file in https://github.com/nod-ai/shark-ai/pull/765.